### ISSUE TITLE
cdo: adding gepotential height fails for files with less than 137 levels

### ISSUE THREAD

**Issue by gisi90** [Open-MSS/data-retrieval#24]
happend for:
levelist=61/to/137

which is a commonly used to reduce data size in combination with high grid resolution (e.g. 0.15deg) and if only lower levels (<16km) are of interest.

cdo fails with:

Adding geopotential height...
cdo    gheight: z-axis 2 has only 77 of 137 hybrid sigma pressure levels!

cdo    gheight (Abort): No 3D variable with hybrid sigma pressure coordinate found!

**Comment by joernu76** [comment]
Mmmh. the convert.sh script reduces the resolution of the ml file *after* processing.
```
ncks -O -d lev,0,0 -d lev,16,28,4 -d lev,32,124,2 $mlfile $mlfile
```
This certainly needs to be aligned with retrieved model levels!

to reduce the need for data transfer. Doesn't 61/to/137 discard the topmost 61 levels? I.e. everything above 16km? Is this good, e.g. for ALIMA?

Either way I do not see why cdo should not be able to compute gph. Are you aware of a standard tool to compute it? We can add that computation easily to add_ancillary, but I'd like to no use "my own" algorithms as much as possible.

**Comment by joernu76** [comment]
Fixed  by #26
